Frontend/student-hub/src/pages/CoursePage.js

```diff
@@ -42,28 +43,33 @@ function CoursePage() {
     const [emojis, setEmojis] = useState([])
     const { setError } = useError();
 
-
+// delay loading state to 1 minute
+const delayTime = 60000;
 // emoji
     useEffect(() => {
         const fetchEmojis = async () => {
+            setIsLoading(true)
             try{
                 const response = await fetch(`${process.env.REACT_APP_API}/emoji`);
                 if (!response.ok) {
                     setError(error.message)
-
                 }
                 const emojistData = await response.json();
                 setEmojis(emojistData);
+
             } catch (error) {
                 setError(error.message)
             }
+            setTimeout(() => {
+                setIsLoading(false)
+                }, delayTime);
```

Comment:
what does this do?

Reply:
I added the delaytime for the loading state to be visible for a set time because when I load the page it loads which makes it difficult to see the loading state.